src/libstd/net/udp.rs

```diff
@@ -903,6 +903,16 @@ mod tests {
         })
     }
 
+    #[test]
+    fn socket_peer_ip4() {
```

Comment:
Pretty sure `each_ip` gives you both IPv4 and IPv6

Reply:
Yeah, I copied from the previous tests, should I rename both?